vmdk_plugin/sanity_test.go

```diff
@@ -210,36 +212,36 @@ func TestSanity(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	checkTouch(t, c, volumeName, "file_to_touch", clients[0].endPoint)
+	checkTouch(t, c, volName, "file_to_touch", clients[0].endPoint)
 
 	for _, elem := range clients {
-		v := volumeVmdkExists(t, elem.client, volumeName)
+		v := volumeVmdkExists(t, elem.client, volName)
 		if v == nil {
 			t.Fatalf("Volume=%s is missing on %s after create",
-				volumeName, elem.endPoint)
+				volName, elem.endPoint)
 		}
 		if v.Driver != driverName {
 			t.Fatalf("wrong driver (%s) for volume %s", v.Driver, v.Name)
 		}
 	}
 
-	err = c.VolumeRemove(context.Background(), volumeName)
+	err = c.VolumeRemove(context.Background(), volName)
 	if err != nil {
 		t.Fatalf("Failed to delete volume, err: %v", err)
 	}
 
 	for _, elem := range clients {
-		if volumeVmdkExists(t, elem.client, volumeName) != nil {
+		if volumeVmdkExists(t, elem.client, volName) != nil {
 			t.Errorf("Volume=%s is still present on %s after removal",
-				volumeName, elem.endPoint)
+				volName, elem.endPoint)
 		}
 	}
 
 	fmt.Printf("Running parallel tests on %s and %s (may take a while)...\n", endPoint1, endPoint2)
 	// Create a short buffered channel to introduce random pauses
 	results := make(chan error, parallelVolumes)
 	createRequest := types.VolumeCreateRequest{
-		Name:   volumeName,
+		Name:   volName,
```

Comment:
Nice change.
The volume name generation should be also changed in the goroutine below to something like:
> createRequest.Name += strconv.Itoa(idx) + strconv.Itoa(i)